e2e/test/scenarios/question/notebook-data-source.cy.spec.ts

```diff
@@ -435,13 +440,7 @@ describe("issue 28106", () => {
           .findByTestId("scroll-container")
           .as("schemasList");
 
-        // The list is virtualized and the scrollbar height changes during scrolling (metabase#44966)
-        // that's why we need to scroll and wait multiple times.
-        // Test is flaky because of this - that's why there are 3 attempts.
-        for (let i = 0; i < 3; ++i) {
-          cy.get("@schemasList").scrollTo("bottom");
-          cy.wait(100);
-        }
+        scrollAllTheWayDown();
```

Comment:
Scrolling 3 times was sometimes not enough (as evidenced by screenshots and videos from failed runs), so now we're making sure the desired result has been achieved.